Survey/Sources/Presentation/Views/Common/AlertMessage.swift

```diff
@@ -26,7 +27,10 @@ extension AlertMessage {
 
     init(error: Error) {
         title = "Error"
-        let message = error.localizedDescription
+        var message = error.asAFError?.underlyingError?.localizedDescription ?? ""
+        if let sError = error as? SError {
+            message = sError.detail
+        }
```

Comment:
Might be better to write this way, so asAFError is not call too often.
```suggestion
        var message: String
        if let sError = error as? SError {
            message = sError.detail
        } else {
            message = error.asAFError?.underlyingError?.localizedDescription ?? ""
        }
```

Reply:
Thanks, I fixed at https://github.com/nkhanh44/ic-survey-ios/pull/15/commits/4f2754088096466f141e871cd4a4f6bb09543285